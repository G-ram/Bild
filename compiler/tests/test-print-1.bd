a = {"h", "e" , "y"};
print(a);
