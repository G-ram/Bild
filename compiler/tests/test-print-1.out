0 : h
1 : e
2 : y
